example/template.cpp

```diff
@@ -49,4 +49,4 @@ int main() {
     std::cout << "int " << cfg.inner.really_inner.really_innermost.int_key << std::endl;
     std::cout << "double " << cfg.inner.really_inner.really_innermost.double_key << std::endl;
     std::cout << "bool " << cfg.inner.really_inner.really_innermost.bool_key << std::endl;
-}
```

Comment:
I haven't had my coffee yet this morning but this looks the other way around.
You're removing the existing newline.
Except I checked in my code and there isn't one.

Reply:
My code is "removing" the `}` without a newline at the end of it, and "adding" `}\n`. Confused yet? LOL

Suffice it to say that my code is doing what it's supposed to, even if GitHub doesn't show it very intuitively.